Worktrolly/worktrolly
# Dev/error

### Functionality:
Throw error whenever there is an internal or external error.

### Solution:
Add Error Component, Error Handler Service and error xml

### Risk level:

### How to test:

- ng serve
- firebase emulators: start
- Press Submit button without filling input fields.
- Then you can see the Error.

### ScreenShots:

![image](https://user-images.githubusercontent.com/54079190/107417584-c78bc300-6b3b-11eb-8aaf-9d01c442c8ed.png)

![image](https://user-images.githubusercontent.com/54079190/107417690-de321a00-6b3b-11eb-9f60-8fcd8496e6a6.png)
